### DISCUSSION TITLE
Does this API still work?

### DISCUSSION THREAD

**Discussion by DaveSkender** [sanko/Robinhood#96] [Q&A] [answered]
[newbie question for this API]
I've not seen any [official API documentation](https://docs.robinhood.com/) for Robinhood beyond their new Crypto stuff.  Did they move everything from `api.robinhood.com` to `trading.robinhood.com/api/v1` or is this access method essentially dead to the world?

**Comment by DaveSkender** [comment, marked as answer]
@sanko, I'm not sure how to interpret the _closing of this question_ without an answer; but will assume you had a good reason for not wanting answering it -- or, just clicked the wrong button.  I'm assuming "Yes", it's dead.

**Reply by bipindra** [reply]
1 sanko may not answer.
2 its http based so you can find out and reverse engineer all api calls.

**Reply by DaveSkender** [reply]
Thanks.  I’m not going to attempt to hack into their API, but even without a direct answer, you’ve answered my question about the state of this project and the openness of the Robinhood API.
